.github/workflows/build.yml

```diff
@@ -30,3 +30,16 @@ jobs:
       with:
         branch: gh-pages # The branch the action should deploy to. It should be any branch other than `main`
         folder: dist # The folder the action should deploy.
+
+    - name: Publish ✨
```

Comment:
That's a good question! I don't know the best practices here, so you tell me.
I chose to make it a 'step' for a couple of reasons:
- if I were to make it a job I would have to do 3/4 exact same steps from build
- it was just faster to try it out

Reply:
From some quick Googling, folks advocate for "keeping jobs simple," but there doesn't seem to be a ton of best-practices out there.

I'm curious if it would get rid of the need to repeat steps if you made Publish a separate job, but used [the `needs` directive](https://docs.github.com/en/actions/using-workflows/about-workflows#creating-dependent-jobs) to specify that the Publish job depends on the Build job?

If so, that feels cleaner to me.

Reply:
I like it